stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/Context.kt

```diff
@@ -67,14 +67,16 @@ internal fun Context.createStreamThemeWrapper(uiMode: UiMode = ChatUI.uiMode): C
     val originalConfig = resources.configuration
 
     return when {
-        this.theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) ->
-            ContextThemeWrapper(this, R.style.StreamUiEmptyTheme)
         this.theme.resolveAttribute(R.attr.streamUiTheme, typedValue, true) ->
```

Comment:
What was the issue here? I checked yesterday and the issue was with GalleryActivity struggling to find basic material attrs. Haven't `StreamUi.AttachmentGallery` already extended material theme?

Reply:
We are substituting the material theme with this `R.style.StreamUiEmptyTheme` here. I thought it would keep all the attrs of the previous theme, but that is not the case.